apps/docs/pages/guides/database/postgres/row-level-security.mdx

```diff
@@ -84,9 +84,13 @@ on profiles
 for select using ( auth.uid() = user_id );
 ```
 
+[comment]: <> (TODO: Improve the RLS docs)
+
```

Comment:
```suggestion
```

Let's not leave comments in here for readability, if there are outstanding issues to address we can make an issue